### ISSUE TITLE
Complier Problems with Chrome and Edge

### ISSUE THREAD

**Issue by JB395** [spacemanholdings/qmix#26]
Complier loading, display and selection not working for current versions of Chrome and Edge

Note that ad blocking is disabled and these browsers support the compilers perfectly for remix.ethereum.org.

Loading the latest local compiler version and showing the list of compilers on Qmix broke between Chrome version 63 and version 64 (64.0.3282.140 on 2018-02-06). Enabling ads and popups makes no difference on versions 64 and later. One highlighted update for Chrome v64 was (https://developers.google.com/web/updates/2018/01/nic64#popup-blocker):

> Improved Pop-up Blocker
> I hate tab-unders. You know them, it’s when a page opens a pop-up to some destination AND navigates the page. Usually one of them is an ad or something that you didn’t want.
> Starting in Chrome 64, these type of navigations will be blocked, and Chrome will show some native UI to the user - allowing them to follow the redirect if they want.
> 

Full list of Chrome v64 changes at https://chromium.googlesource.com/chromium/src/+log/63.0.3239.84..64.0.3282.119

Some possibly related fixes for Remix are:

Move compiler to remix solidity #1027 https://github.com/ethereum/remix-ide/pull/1027

Fix loading compiler #1169 https://github.com/ethereum/remix-ide/pull/1169

Merge pull request #1169 from ethereum/fixCompilerLoading https://github.com/ethereum/remix-ide/commit/3fb2c1c78c76bf86ba0f4914e762bb2786e22097

Update package.json https://github.com/ethereum/remix-ide/commit/3778c622079e9b0d35ed26f03ca319e1ca071061

**Comment by spacemandev-git** [comment]
Hmm, I'm unable to replicate, on Chrome v70 the compiler list seems to load perfectly fine. Maybe try updating Chrome?

**Comment by JB395** [comment]
No, endless loop of "Loading Compiler". It is probably going to work on iOS, but are you on a PC with Win 10? I have tried across multiple PCs with Win 10 and none work with any version of Chrome > 63. 

![screenhunter 2232](https://user-images.githubusercontent.com/29760787/52683235-35329380-2f10-11e9-805b-220118bd41df.jpg)
